awx/main/access.py

```diff
@@ -979,7 +984,7 @@ def can_use(self, obj):
     def can_change(self, obj, data):
```

Comment:
`CredentialAccess` has a `can_add` method which remains unchanged here. See:

```python
         if data and data.get('organization', None):
             organization_obj = get_object_from_data('organization', Organization, data)
             return check_user_access(self.user, Organization, 'change', organization_obj, None)
```

This would seem to ultimately reference organization admin_role, not the organization credential admin role.